modules/ecs_service_fargate_elb/variables.tf

```diff
@@ -86,3 +88,32 @@ variable "target_group_arn" {
   description = "Target of an Amazon ALB (Load Balancer), as given by a load balancer."
 }
 
+variable "datadog_api_key" {
+  description = "Datadog API key. This will activate the Datadog agent sidecar/replica container. To accommodate additional load, please increase var.cpu to an additional 512. Similarly, increase var.memory to an additional 1024."
+  default     = ""
+  type        = string
+}
+
+variable "datadog_agent_version" {
+  description = "Datadog agent container version."
+  default     = "latest"
+  type        = string
+}
+
+variable "datadog_agent_cpu" {
+  description = "Datadog agent container cpu. Units are 1024 = 1 vCPU."
+  default     = 256
```

Comment:
@rafops The comment above says that var.cpu is supposed to be increased by 512, but this is only 256. Is this inconsistent or am I missing something? Apologies if I am missing something!

Reply:
@sskirby The description of `datadog_api_key` says you may need to increase `var.cpu` (`variable "cpu"`) to an additional 256. The default of `256` is for a different variable `var.datadog_agent_cpu` (`variable "datadog_agent_cpu"`)

Reply:
@rafops I thought the reason for increasing the `var.cpu` by 512 was because the Datadog sidecar needed 512. Here it looks like Datadog sidecar only needs 256.